### ISSUE TITLE
Copy aria-* attributes to textarea

### ISSUE THREAD

**Issue by vanowm** [WebCoder49/code-input#52]
For accessibility compatibility `aria-*` attributes should be copied to the `textarea`, in fact maybe almost all attributes should be copied too?
